07-On demand scan as an API/README.md

```diff
@@ -0,0 +1,312 @@
+- [API to get control scan result](README.md#API-to-get-control-scan-result)
+- [Feedback](README.md#Feedback)
+
+-----------------------------------------------------------------
+## Overview 
+The Azure Tenant Security Solution (AzTS) provides APIs for users to allow on demand scan for a subscription and get control scan result.
+
+This document will walk you through:
+1. Register an application in Azure AD to represent a client application.
+2. Configure permissions for WebAPI app registration.
+3. Get administrator consent for WebAPI app registration.
+4. Generate authentication token to access subscriptions.
+5. Use the access token to call API endpoints.
+
+## Prerequisites (For AzTS admin)
+Admin has to enable the flag **FeatureManagement__OnDemandScanAPI** in order to provide access over API endpoints.
+How to enable 'OnDemandScanAPI' from Azure Portal:
+1. Go to Azure Portal.
+2. Go to **Resource Groups**.
+3. Select your Resource Group where you have configured AzTS set up.
+4. Select the App Service for API 'AzSK-ATS-API-xxxxx'.
+5. Go to **Configuration**.
+6. Set **FeatureManagement__OnDemandScanAPI** as true.
+7. Save.
+
+## Step 1: Register an application in Azure AD to represent a client application
+Before generating the token or making an API call, client app registration needs to be created.
+</br>
+Follow below steps to create client application:
+1. Go to the Azure portal to register your application.
+
+2. Search for and select **App registrations**.
+
+3. Select **New registration**.
+
+4. When the **Register an application** page appears, enter your application's registration information:
+
+    - In the **Name** section, enter a meaningful application name that will be displayed to users of the app, such as client-app.
+
+    - In the **Supported account types** section, select **Accounts in any organizational directory (Any Azure AD directory - Multitenant)**.
+
+5. In the **Redirect URI** section, select 'Web' and leave the URL field empty for now.
+
+6. Select **Register** to create the application.
+
+7. On the app **Overview** page, find the **Application (client) ID** value and record it for later.
+
+8. Create a client secret for this application to use in a subsequent step (only if you want to use 'Client credential flow' to generate the token).
+
+    - From the list of pages for your client app, select **Certificates & secrets**, and select **New client secret**.
+
+    - Under **Add a client secret**, provide a **Description**. Choose when the key should expire, and select **Add**.
+
+9. When the secret is created, note the key value for use in a subsequent step.
+
+</br>
+
+[Back to top…](README.md#On-this-page)
+
+<!-- TBD: This token is not access subs, but to access AzTS REST API endpoints -->
+
+## Step 2: Configure permissions for WebAPI app registration
+1. Go to Azure Portal.
+2. Go to **App Registration**.
+3. Select your WebAPI App Registration.
+4. Go to **API Permissions**.
```

Comment:
You can add screenshots. Also, this step is to be performed by the admin.

Reply:
Screenshot added